config/servicemappings/vertexai.yaml

```diff
@@ -64,7 +64,7 @@ spec:
     - name: google_vertex_ai_endpoint
       kind: VertexAIEndpoint
       idTemplate: "projects/{{project}}/locations/{{region}}/endpoints/{{name}}"
-      idTemplateCanBeUsedToMatchResourceName: false
+      idTemplateCanBeUsedToMatchResourceName: true
```

Comment:
Is this change on purpose? This property/field is usually set to true when we want to enable import/export tool for this resource. Or is it configured because we want to use export for vertex AI resources in e2e tests?

Reply:
I am not familiar with this field. But according to #2135, if this field is set to `false`, test `TestNewFromURI` will skip on the resource?

Reply:
You are right. And the field was designed for the export/import tool: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/pkg/apis/core/v1alpha1/servicemapping_types.go#L88